specification/loadtestservice/resource-manager/Microsoft.LoadTestService/stable/2022-12-01/loadtestservice.json

```diff
@@ -0,0 +1,963 @@
+          },
+          {
+            "in": "body",
+            "name": "LoadTestResourcePatchRequestBody",
+            "description": "LoadTest resource update data",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/LoadTestResourcePatchRequestBody"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- resource updated",
+            "schema": {
+              "$ref": "#/definitions/LoadTestResource"
+            }
+          },
+          "202": {
+            "description": "Accepted -- LoadTest resource patch request accepted"
+          },
+          "default": {
+            "description": "Resource provider error response about the failure.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "LoadTests_Update": {
+            "$ref": "./examples/LoadTests_Update.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "LoadTests"
+        ],
+        "description": "Delete a LoadTest resource.",
+        "operationId": "LoadTests_Delete",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/LoadTestNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- resource deleted"
+          },
+          "202": {
+            "description": "Accepted -- LoadTest resource delete request accepted"
+          },
+          "204": {
+            "description": "No Content -- resource deleted"
+          },
+          "default": {
+            "description": "Resource provider error response about the failure.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "LoadTests_Delete": {
+            "$ref": "./examples/LoadTests_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.LoadTestService/loadTests/{loadTestName}/outboundNetworkDependenciesEndpoints": {
```

Comment:
>outboundNetworkDependenciesEndpoints [](http://example.com/codeflow?start=133&length=36)

Consider outboundNetworkDependencyEndpoints. Multiple plurals is confusing because it implies the collection includes two different kinds of things.

Reply:
Can't change as we're following the recommendation for VNet injection (category:: Freeform) - https://armwiki.azurewebsites.net/rpaas/contract/resourcetypecategory.html?q=category @mentat9